en/orm/saving-data.rst

```diff
@@ -874,8 +874,8 @@ the singular, underscored version of the association name. For example::
 Saving HasOne Associations
 --------------------------
 
-When saving hasOne associations, the ORM expects a single nested entity at the
-singular, underscored version of the association name. For example::
+When saving hasOne associations, the ORM expects a single nested entity named with the
+singular, lowercase version of the association name. For example::
```

Comment:
@ravage84 do you mean a central section on this same page?
I guess this could be another way to do it; though it would add a bit more content to what's an already long page.
The bottom line is that I'm not particularly favoring a specific way to do this; just trying my best to clarify it for everyone. Whatever everyone else thinks is best will work for me, as long as we make it more explicit.

The reason is that being a beginner in Cake, working on my first project, I have the particular "advantage" of maybe seeing faster the places where the documentation seems harder to understand or laconic.
One of the main ones for me has been the fact that many little rules or details such as the above, seem to be half-inferred from the examples provided in the manual and not always very explicitly stated in the text (which makes it harder to apply them in code directly as I feel quite often that I have to "reverse-engineer" the examples in order to make sure I got everything right and not missing something).
That's what I'm trying to improve for others, as I go along and digest for myself this huge thing... :)